  Source.cpp
C:\Qt\Projects\FSDM_12011_ru\OOP\Lesson_05_03_21\Source.cpp(71,2): warning C4286: 'const MyException *': is caught by base class ('const std::exception *') on line 66
  Lesson_05_03_21.vcxproj -> C:\Qt\Projects\FSDM_12011_ru\OOP\Lesson_05_03_21\Debug\Lesson_05_03_21.exe
